university-task-02/src/pages/Home/components/Tooltip/Tooltip.jsx

```diff
@@ -0,0 +1,17 @@
+import React, { useState } from 'react';
+import './Tooltip.css'; 
+
+const Tooltip = ({ children, text }) => {
+  const [isHovered, setIsHovered] = useState(false);
```

Comment:
Do you really need a state here to implement the hover effect? Why don't you use CSS hover functionality to show/hide the tooltip?